core/src/test/kotlin/com/linecorp/kotlinjdsl/querydsl/where/QueryDslImplWhereTest.kt

```diff
@@ -67,8 +67,7 @@ internal class QueryDslImplWhereTest : WithKotlinJdslAssertions {
         val actual = QueryDslImpl(Data1::class.java).apply {
             select(distinct = true, Data1::class.java)
             from(entity(Data1::class))
-            where(predicateSpec1)
-            where(predicateSpec2)
+            where(predicateSpec1,predicateSpec2)
```

Comment:
-- korean
where 메소드는 predicates를 받을 수 없기 때문에 잘못된 테스트로 보입니다.

-- english
The test appears to be an invalid test because where method cannot receive predicates as parameter.

Reply:
-- korean
where 메소드에 대한 검증은 총 6개로 이루어져야 할 것으로 보입니다.
`where`, `where - null`, `whereAnd - vararg`, `whereAnd - list`, `whereOr - vararg`, `whereOr - list`

-- english
It seems that there should be a total of six tests for the where method.
`where`, `where - null`, `whereAnd - vararg`, `whereAnd - list`, `whereOr - vararg`, `whereOr - list`

Reply:
Code formatting seems to be necessary as well.
```kotlin
            where(predicateSpec1, predicateSpec2)
```

Reply:
@shouwn 말씀하신 테스트 케이스들 추가하였습니다.
[feature: add whereAnd and whereOr method](https://github.com/line/kotlin-jdsl/pull/75/commits/35b281a14575ebcbc56bcf1a271cc2b1924db6f8)

Reply:
@cj848 [refactor: rollback wheres() testCase](https://github.com/line/kotlin-jdsl/pull/75/commits/384ef1c71608fdd299aa3bceeb64b4297ece583d)에 해당 내용 적용하였습니다. 현재는 다른 메소드로 변경되었습니다.